cogboard-app/src/test/kotlin/com/cognifide/cogboard/logStorage/model/QuarantineRuleTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.cognifide.cogboard.logStorage.model
+
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
+
+class QuarantineRuleTest {
+    @Test
+    fun `Properly parses correct rule`() {
+        val rule = QuarantineRule.from(JsonObject("{\"label\": \"Example label\", \"reasonField\": \"Reason\", \"regExp\": \"^a\", \"checked\": true}"))
+        assertEquals("Example label", rule?.label)
+        assertEquals("Reason", rule?.reasonField)
+        assertEquals("^a", rule?.regex?.pattern)
+        assertEquals(true, rule?.enabled)
```

Comment:
rule cannot be null `rule?.` -> `ru`le.`

Reply:
Done.